python/mxnet/model.py

```diff
@@ -212,7 +212,11 @@ def _train_multi_device(symbol, ctx, arg_names, param_names, aux_names,
     for epoch in range(begin_epoch, end_epoch):
         # Training phase
         tic = time.time()
```

Comment:
I would recommend always make eval_metric a list, and do special transformation if it is not in the beginning